dev/image/Dockerfile

```diff
@@ -2,7 +2,10 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM gitpod/workspace-full:latest
+# if you want to run this image in a workspace, follow these steps:
+# gcloud auth configure-docker europe-docker.pkg.dev
+# gcloud auth login --no-launch-browser
+FROM europe-docker.pkg.dev/gitpod-artifacts/docker-dev/workspace-base-images:full
```

Comment:
We need to explicitly test the leeway cache, running a full build of Gitpod, without any prior cache, to assert that we're not missing anything.

`-c` flag should help with this, refer to the build.ts as an example to see current usage (which relies on cache).

Reply:
Hey @princerachit in talking with @csweichel he recommended that we run a build with caching disabled in `Leeway`. 

To do this I:
1. patched `./.werft/build.ts` in the five spots where `leeway build` is called to include `-c none`. Four spots were using `-c remote`, one did not specify `c` or `caching` at all.
2. sideloaded the updated build.ts (using no cache) file as part of the werft job like so `werft run github -j ./.werft/build.yaml -s ./.werft/build.ts `

Good news, and bad news! 

It appears to have caught an issue, which requires research.

[This build timed out](https://werft.gitpod-dev.com/job/gitpod-build-prs-test-dazzle-rewrite.86), can you check to see why, and if the test is still valid? I suspect the test is starting too soon, and may need its timeout increased, or to be delayed somehow.

We'll have to resolve this prior to accepting the PR and beginning dog-fooding.

Reply:
I tried rerunning the tests. Build [failed](https://werft.gitpod-dev.com/job/gitpod-build-prs-test-dazzle-rewrite.90/logs#build:components/ide/code:docker) due to nodejs/yarn error and then it [succeeded](https://werft.gitpod-dev.com/job/gitpod-build-prs-test-dazzle-rewrite.91) on rerun. The issues doesn't seem to be related to the PR. I have reached out to the webapp team to seek more information.

Reply:
Nice! I see you increased the timeout too. Thank yous.